### ISSUE TITLE
Vertical bar are broken in Chart/Gears activities on Windows

### ISSUE THREAD

**Issue by llaske** [llaske/sugarizer#1646]
On Chrome Window, the vertical bar used in activities toolbar to split group of icons is broken.

It look like:

![image](https://github.com/llaske/sugarizer/assets/981841/8dda00bb-319e-4819-b09c-52a59fdb9352)

and should be:

![image](https://github.com/llaske/sugarizer/assets/981841/a0585134-2eeb-4681-8afb-897dbe254b6b)

Issue is both on Chart and Gears activity

**Comment by hharshas** [comment]
yeah, actually the issue persists in one of the the other activity also (I don't remember it clearly) but the problem is in file sugarweb lib's sugar-200dpi.css file with the hr tag rather than these particular activities.

**Comment by llaske** [comment]
@hharshas do you know a way to fix it to ensure it will works in all environments?

**Comment by hharshas** [comment]
Nothing comes to my mind easier than changing the "hr" tag's style in all of the activities's lib folder manually, so we can avoid this error in the future use of <hr> inside the "toolbar" classes. @llaske I guess if you are aware of any other way to make changes at one place and it will automatically update all sugarweb library's style files, that may come in handy.

**Comment by llaske** [comment]
> I guess if you are aware of any other way to make changes at one place and it will automatically update all sugarweb library's style files, that may come in handy.

I've got script to do the job to copy a file changed in sugarweb in every activity.
My question was more: could we find a simple implementation of the `hr` tag that will work in any environment (any OS any browser)?

**Comment by hharshas** [comment]
@llaske, I need to spend some time going through the code base and other things. I'm quite busy with my internship, but if the issue remains unresolved for a while, I'll make sure to look into it.

**Comment by MdSaifAliMolla** [comment]
@llaske fixed it.  #1654

**Comment by AdityaKrSingh26** [comment]
Hey @llaske could you please have a look at my pr (#1673)

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1673
